src/app/core/components/content-top-bar/content-top-bar.component.ts

```diff
@@ -0,0 +1,15 @@
+import { Component, Input } from '@angular/core';
+
+@Component({
+  selector: 'alg-content-top-bar',
+  templateUrl: './content-top-bar.component.html',
+  styleUrls: [ './content-top-bar.component.scss' ],
+})
+export class ContentTopBarComponent {
+  @Input() fullFrameContent?: boolean;
+  @Input() currentMode?: string;
+  @Input() currentContent?: any;
+  @Input() scrolled?: boolean;
+
+  onEditCancel(): void {}
```

Comment:
Bug: we cannot leave edit mode

Can be reproduced on [this group](https://dev.algorea.org/branch/feature/reorganize-top-bar/en/#/groups/by-id/672913018859223173;path=52767158366271444/edit) or [this item](https://dev.algorea.org/branch/feature/reorganize-top-bar/en/#/activities/by-id/1199335416413275996;path=4702,4102,128139237432513103;attempId=0/edit)

Reply:
Fixed